mri/operators/linear/wavelet.py

```diff
@@ -125,10 +126,10 @@ def op(self, data):
             )
             coeffs = np.asarray(coeffs)
         else:
-            coeffs, self.coeffs_shape = self._op(data)
+            coeffs, self.coeffs_shape, self.coarse = self._op(data)
```

Comment:
Are there not parts of the code which depend on getting all the coefficients?

Reply:
Hmmm not from what I recall. Let me add another method to get the coefficients anyway.
(esp currently we only return flattened versions)